megamek/src/megamek/server/totalwarfare/TWGameManager.java

```diff
@@ -8920,7 +8933,7 @@ else if ((waterDepth > 0)
                 && !(entity.getMovementMode() == EntityMovementMode.HOVER)) {
             PilotingRollData waterRoll = entity.checkWaterMove(waterDepth, entity.moved);
             if (waterRoll.getValue() != TargetRoll.CHECK_FALSE) {
-                doSkillCheckInPlace(entity, waterRoll);
+                doSkillCheckInPlace(entity, waterRoll, vPhaseReport);
```

Comment:
maybe call the variable vPhaseReport - displacementReport so that it is clear this in a internal version of the report